src/components/Comment/Comment.tsx

```diff
@@ -384,12 +420,30 @@ export const Comment = ({
                       </button>
                     )}
                 </div>
-                <div>
+                <Row>
+                  {/* You can't mute unless logged in and you can't yourself */}
+                  {user && comment.userProfile.userId !== user.userId ? (
+                    <div className={buttonHeightOverrides}>
```

Comment:
the variations between this and `AbuseReportForm` cause the exact position to be slightly off, and for the on highlight to be different.

<img width="124" alt="Screenshot 2020-04-02 at 13 09 23" src="https://user-images.githubusercontent.com/8831403/78247733-48713b80-74e3-11ea-93e5-3a7dfec27894.png">
<img width="103" alt="Screenshot 2020-04-02 at 13 09 28" src="https://user-images.githubusercontent.com/8831403/78247738-4a3aff00-74e3-11ea-999a-6ad5a7d86630.png">

This should be fixed in another PR by changing `AbuseReportForm` to apply src foundations as it is due to src-foundations

Reply:
Good spot. Agreed, lets move the abuse form over to src as well, and maybe do the same for Reply, Pick and Share at the same time?

Reply:
yes agreed!